### ISSUE TITLE
Unexpected Performance of L=2 Model in Liquid Water Training

### ISSUE THREAD

**Issue by cesaremalosso** [ACEsuit/mace#858]
Hi,  

I've been training a model for liquid water using the following parameters, varying `L` between 1 and 2:  

```bash
srun -n 4 mace_run_train \
    --name="L1_3" \
    --train_file="../../dataset/train_configs.xyz" \
    --valid_file="../../dataset/test_configs.xyz" \
    --test_file="../../dataset/test_configs.xyz" \
    --config_type_weights='{"Default":1.0}' \
    --model="MACE" \
    --E0s='{1:-13.4131335925908, 8:-429.4668907468319}' \
    --max_L=1 \
    --r_max=5.0 \
    --num_interactions=2 \
    --correlation=3 \
    --max_ell=3 \
    --valid_batch_size=4 \
    --batch_size=4 \
    --max_num_epochs=200 \
    --energy_key='energy' \
    --forces_key='forces' \
    --swa \
    --start_swa=160 \
    --ema \
    --ema_decay=0.99 \
    --amsgrad \
    --num_workers=1 \
    --restart_latest \
    --device=cuda \
    --seed=3926 \
    --distributed
```

I trained four equivalent models with different seeds for both `L=1` and `L=2`, selecting the best-performing model in each case. However, the results are unexpected:  

### **Best L=1 Model Performance:**  
```
2025-03-12 01:32:41.159 INFO: Error-table on TRAIN and VALID:
+---------------+---------------------+------------------+-------------------+
|  config_type  | RMSE E / meV / atom | RMSE F / meV / A | relative F RMSE % |
+---------------+---------------------+------------------+-------------------+
| train_default |            2.0      |          9.7     |          0.51     |
| valid_default |            0.2      |          6.7     |          0.82     |
+---------------+---------------------+------------------+-------------------+
```

### **Best L=2 Model Performance:**  
```
2025-03-12 10:00:34.345 INFO: Error-table on TRAIN and VALID:
+---------------+---------------------+------------------+-------------------+
|  config_type  | RMSE E / meV / atom | RMSE F / meV / A | relative F RMSE % |
+---------------+---------------------+------------------+-------------------+
| train_default |            1.8      |          9.2     |          0.49     |
| valid_default |            0.7      |          6.3     |          0.77     |
+---------------+---------------------+------------------+-------------------+
```

While `L=2` performs slightly better for forces, it is noticeably worse for energy, especially on the validation set. I was expecting `L=2` to yield improved accuracy overall.  

Do you have any insights into why this might be happening? 

Best,  
Cesare
